test/test_per_overload_api.py

```diff
@@ -60,5 +62,16 @@ def test_basics_opoverload(self):
 
         self.assertRaises(RuntimeError, lambda: add_tensoroverload(a, a, out=b))
 
+        add_outoverload = add_packet.out
+        self.assertEqual(str(add_outoverload.inputs), "[Tensor self, Tensor other, Scalar alpha, Tensor& out]")
+        self.assertTrue(add_outoverload.inputs[3].is_mutable)
+        self.assertTrue(not add_outoverload.inputs[2].is_mutable)
+        self.assertTrue(add_outoverload.inputs[3].kwarg_only)
+        self.assertTrue(add_outoverload.inputs[2].has_default_value)
+        self.assertEqual(add_outoverload.inputs[2].default_value, 1)
+        self.assertEqual(add_outoverload.inputs[2].name, "alpha")
+        self.assertEqual(add_outoverload.inputs[2].type, torch._C.NumberType.get())
```

Comment:
ideally we wouldn't want these Type class objects to have their module set to `torch` instead of `torch._C` but changing this would be bc breaking for people working with TorchScript graphs (these types are being used their -- see `test_jit.py`)